docs/source/how-to/migrations/upgrade-to-16.md

```diff
@@ -1,23 +1,6 @@
 # Upgrading your OAuthenticator to version 16.0
 
-The following section describes what to keep in mind when upgrading to OAuthenticator 16.0.
-
-````{important}
-(token_request)=
-
-```{rubric} The token request
-```
-Whenever _token request_ is used, it refers to the HTTP request exchanging the [oauth code](https://www.rfc-editor.org/rfc/rfc6749#section-1.3.1) for the [access token](https://www.rfc-editor.org/rfc/rfc6749#section-1.4).
-
-This request is sent to the {attr}`.OAuthenticator.token_url` in {meth}`.OAuthenticator.get_token_info` method.
-
-(userdata_request)=
-```{rubric} The userdata request
-```
-Whenever _userdata request_ is used, it refers to the HTTP request that's exchanging the the `access token` to get the `userdata`.
-
-This request is sent to {attr}`.OAuthenticator.userdata_url` in {meth}`.OAuthenticator.token_to_user` method.
-````
+The following section describes what to pay attention to when upgrading to OAuthenticator 16.0.
 
 ## Deprecations
```

Comment:
The title is deprecations which makes me thing that various things still work but is going to stop working. Is this the case, or is it just clean breaking changes, or a mix of breaking changes and deprecations?

It seems that part 1 and 2 are deprecations, but part 3 is a breaking change. Since part 1 and 2 is mentioned to be deprecations, which still could fall under a category of "breaking changes" in my mind, what do you think about changing the title to "Breaking changes" or similar?

![image](https://user-images.githubusercontent.com/3837114/216338553-a75f0ec3-49f1-4cda-bfe5-6fdf1c59c141.png)

Reply:
Makes sense! Thanks @consideRatio